cdap-docs/admin-manual/source/installation/security.rst

```diff
@@ -270,43 +286,58 @@ LDAP Authentication
+                                                           ``server.``
+                                                           ``LDAPLoginModule``
+``security.authentication.handler.debug``                  ``false``                   Set to ``true`` to enable debugging
+``security.authentication.handler.hostname``               ``<hostname>``              LDAP server host
+``security.authentication.handler.port``                   ``<port>``                  LDAP server port
+``security.authentication.handler.userBaseDn``             ``<userBaseDn>``            LDAP base Distinguished Name to
+                                                                                       authenticate with
+``security.authentication.handler.userRdnAttribute``       ``<userRdnAttribute>``      LDAP attribute used as the relative 
+                                                                                       Distinguished Name
 ``security.authentication.handler.userObjectClass``        ``<userObjectClass>``
-========================================================== ==================================================================
+========================================================== =========================== ======================================
 
 In addition, you may configure these optional properties in ``cdap-site.xml``:
 
-========================================================== ==================================================================
-Property                                                   Value
-========================================================== ==================================================================
-``security.authentication.handler.bindDn``                 ``<bindDn>``
-``security.authentication.handler.bindPassword``           ``<bindPassword>``
-``security.authentication.handler.userIdAttribute``        ``<userIdAttribute>``
-``security.authentication.handler.userPasswordAttribute``  ``<userPasswordAttribute>``
-``security.authentication.handler.roleBaseDn``             ``<roleBaseDn>``
-``security.authentication.handler.roleNameAttribute``      ``<roleNameAttribute>``
-``security.authentication.handler.roleMemberAttribute``    ``<roleMemberAttribute>``
-``security.authentication.handler.roleObjectClass``        ``<roleObjectClass>``
-========================================================== ==================================================================
+========================================================== =========================== ======================================
+Property                                                   Default Value               Description
+========================================================== =========================== ======================================
+``security.authentication.handler.bindDn``                 ``<bindDn>``                The user on an external LDAP server 
```

Comment:
"The Distinguished Name used to bind to the LDAP server and search the directory"